crates/astria-sequencer/justfile

```diff
@@ -0,0 +1,34 @@
+default:
+  @just --list
+
+set dotenv-load
+set fallback
+
+default_env := 'local'
+copy-env type=default_env:
+  cp {{ type }}.env.example .env
+
+run:
+  cargo run
+
+
+run-cometbft:
+  cometbft init
+  sed -i'.bak' 's/timeout_commit = "0s"/timeout_commit = "15s"/g' ~/.cometbft/config/config.toml
```

Comment:
I am a bit uncomfortable changing people's local files. What if they have a cometbft already present?

Reply:
Would be open to alternate scripts - this is just what I rebased from main

Reply:
Yeah, good point. Setting up and taking down the env is certainly more complex than a just script allows. Maybe add a disclaimer?